assets/js/modules/ads/components/settings/SettingsForm.js

```diff
@@ -145,6 +147,7 @@ export default function SettingsForm() {
 							}
 						) }
 					</ConversionTrackingToggle>
+					{ fpmEnabled && <FirstPartyModeToggle /> }
```

Comment:
Rendering the toggle in the settings edit view is actually out of scope for this issue, it's due in https://github.com/google/site-kit-wp/issues/9655.

We _can_ keep it here, but I'd suggest removing it so it can be implemented and QA'd as part of the above issue.

Reply:
Removed, thank you!